cartographer_ros/cartographer_ros/node.h

```diff
@@ -214,6 +214,7 @@ class Node {
 
   // These are keyed with 'trajectory_id'.
   std::map<int, ::cartographer::mapping::PoseExtrapolator> extrapolators_;
+  std::map<int, ::ros::Time> last_published_tf_stamp_;
```

Comment:
Please use plural for consistency with the other maps here. Rename to `last_published_tf_stamps_`.